### PR TITLE
Git Attributes

### PR DESCRIPTION
**Pull request by 7h3kk1d**
Includes test commit do not merge

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -2,3 +2,7 @@
 # decides that the content is text, its line endings are converted to LF on
 # checkin. When the file has been committed with CRLF, no conversion is done.
 * text=auto
+
+*.ml linguist-generated
```
Comment (cyrus-) [inline .gitattributes]: maybe leave a comment here to https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github to explain what this is doing

Comment (cyrus-) [reply .gitattributes]: This is a quirk of the fact that we are using Reason for everything other than autogenerated stuff, so maybe better to explicitly list the auto-generated files/directories -- Init.ml and the exercises directory I think are the culprits?

**Review by cyrus- — CHANGES_REQUESTED** [review]

**Review by cyrus- — APPROVED** [review]

**Comment by cyrus-** [comment]
@7h3kk1d lgtm now, is this ready to merge?

**Comment by 7h3kk1d** [comment]
> @7h3kk1d lgtm now, is this ready to merge? 

@cyrus- yeah, merge whenever. Thanks!

**Pull request merged**